cypress/fixtures/users.json

```diff
@@ -0,0 +1,232 @@
+[
```

Comment:
These are example fixtures I left in as a placeholder for when we have mocks for integration tests, but it is a good idea to remove them. I do want to keep the directory around if possible, since it does have special meaning to Cypress (the data for API mocks go in `.json` files in the fixtures directory). Should I keep a `.gitignore` in this directory to keep it in git, or perhaps a `README.md` or `Fixtures.md` explaining how Cypress uses this directory?

Reply:
I removed the example files, and wrote up a quick README.md that links to the [Cypress documentation](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Fixture-Files) on how the files that were in the fixtures directory should be used.